### PR TITLE
Clarify the cardinality limit section

### PR DESCRIPTION
**Pull request by MrAlias**
1. Currently the term "view" is overused within the section. It is used to mean the SDK, views, and aggregators. This replaces the term with an appropriate replacement
2. Clarify the priority order for how a cardinality limit is configured within its own new section
3. Create a section for the overflow attributes

### CODE REVIEW

File: specification/metrics/sdk.md [hunk 1]
```diff
@@ -704,19 +706,24 @@ given instrument before starting a subsequent round of collection.
 
 **Status**: [Experimental](../document-status.md)
 
-Views SHOULD support being configured with a cardinality limit to be
-applied to all aggregators not configured by a specific view, specified
-via `MetricReader` configuration.
+SDKs SHOULD support being configured with a cardinality limit. A cardinality
+limit is the hard limits on the number of data points that can be written per
```
Comment (reyang) [inline specification/metrics/sdk.md:710]: "data points" or "metric streams"?

"written" or "collected"?

Cardinality limit is a general concept, I don't think we should _redefine_ it as per aggregation thing (e.g. cardinality limit across the entire SDK would also make sense, even if we don't plan to add it right now).

Comment (MrAlias) [reply specification/metrics/sdk.md:710]: I had just copied this from below, but I agree we can probably clarify. I can change "data points" to "metric streams" and "written" to "collected", both sound more appropriate to me.

Do you have a suggestion you can proposed on how to revise the scope of the cardinality limits?

Comment (MrAlias) [reply specification/metrics/sdk.md:710]: Maybe just:

> A cardinality limit is the hard limit on the number of metric streams that can be collected.

I'll update to this.

Comment (MrAlias) [reply specification/metrics/sdk.md]: Updated. PTAL.

**Review by reyang — APPROVED** [review]

File: specification/metrics/sdk.md [hunk 2]
```diff
@@ -704,19 +706,23 @@ given instrument before starting a subsequent round of collection.
...
+The cardinality limit for an aggregation is defined one of three ways:
 
-The RECOMMENDED default aggregation cardinality limit is 2000.
+1. A [view](#view) with criteria matching the instrument an aggregation is
+   created for has an `aggregation_cardinality_limit` value defined for the
+   stream, that value SHOULD be used.
+2. If there is no matching view, but the `MetricReader` defines a default
+   cardinality limit value based on the instrument an aggregation is created
+   for, that value SHOULD be used.
+3. If none of the previous values are deinfed, the default value of 2000 SHOULD
```
Comment (reyang) [inline specification/metrics/sdk.md:722]:
```suggestion
3. If none of the previous values are defined, the default value of 2000 SHOULD
```

File: specification/metrics/sdk.md [hunk 3]
```diff
@@ -731,16 +737,15 @@ limit, as a result.
...
 #### Synchronous instrument cardinality limits
 
-Views of synchronous instruments with cumulative aggregation
-temporality MUST continue to export the all attribute sets that were
-observed prior to the beginning of overflow.  Metric events
-corresponding with attribute sets that were not observed prior to the
-overflow will be reflected in a single data point described by (only)
-the overflow attribute.
+Aggregators for synchronous instruments with cumulative temporality MUST
+continue to export the all attribute sets that were observed prior to the
```
Comment (reyang) [inline specification/metrics/sdk.md:741]: not introduced by this PR, "the all" sounds weird.

**Review by jsuereth — APPROVED** [review]

**Pull request merged**